lib/ansiblelint/rules/__init__.py

```diff
@@ -245,11 +245,37 @@ def __repr__(self) -> str:
                           for rule in sorted(self.rules, key=lambda x: x.id)])
 
     def listtags(self) -> str:
+        tag_desc = {
+            "behaviour": "Indicates a bad practice or behavior",
+            "bug": "Likely wrong usage pattern",
+            "command-shell": "Specific to use of command and shell modules",
+            "core": "Related to internal implementation of the linter",
+            "deprecations": "Indicate use of features that are removed from Ansible",
```

Comment:
coming in late w/ some comments - deprecation is supposed to mean features that will be removed. Consider changing to  'features that will be removed from Ansible'.